### DISCUSSION TITLE
Is TTS alive?

### DISCUSSION THREAD

**Discussion by larry77** [coqui-ai/TTS#3600] [General Q&A]
Hi am making my first baby steps with this project and I love it. My only concern is that coqui.ai is shutting down, so I wonder if TTS is going to become a fossil. Is the community still committed to its development? Thanks

**Comment by davidmartinrius** [comment]
Even if it is not going to be updated anymore it is one of the most powerful repos for tts yet.

**Reply by larry77** [reply]
I see, but I still hope someone will fork it and go on developing.

**Reply by davidmartinrius** [reply]
I am an active developer. Today I published a new tool for this project https://github.com/coqui-ai/TTS/discussions/3604 There are other developers who are also contributing in their own way.

**Reply by fmailhot** [reply]
Great to hear that people are still developing here. Is there a plan to support recent/current/future Python versions?

**Reply by davidmartinrius** [reply]
Hi @fmailhot You can fork this project update the packages and python by yourself. But wouldn't expect this project to be updated soon... you can see my fork at https://github.com/davidmartinrius/TTS I optimized and improved coquiTTS a bit.
